src/Core/Models/Business/SubscriptionUpdate.cs

```diff
@@ -292,18 +292,18 @@ public override List<SubscriptionItemOptions> UpgradeItemsOptions(Subscription s
     {
         var updatedItems = new List<SubscriptionItemOptions>();
 
-        AddNewSecretsManagerItems(updatedItems);
+        AddNewSecretsManagerItems(updatedItems, subscription);
 
         return updatedItems;
     }
 
-    private void AddNewSecretsManagerItems(List<SubscriptionItemOptions> updatedItems)
+    private void AddNewSecretsManagerItems(List<SubscriptionItemOptions> updatedItems, Subscription subscription)
```

Comment:
⛏️ This `subscription` parameter isn't currently being used. If it's not required for any future work in this method, we can remove it